index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Koshao's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="container-toolbar">
+            <h1 class="toolbar-title">Welcome</h1>
+            <span class="toggle"><img src="./Images//enabled.png" alt="nav"></span>
+        </div>
+    </header>
+
+    <section class="headline">
+        <div class="headline-name-job">
+            <h2 class="headline-name">Hey there, I'm Koshao</h2>
```

Comment:
- [ ] Kindly change this from the `h2` tag to the `h1` tag. This is the main header of the webpage and main headers should be embedded in `h1` tags for semantic purposes. 👍